src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -77,8 +77,27 @@ export const disabilityNameTitle = ({ formData }) => {
 };
 
 
+export const facilityDescription = ({ formData }) => {
+  return (
+    <p>Tell us about facilities where VA treated you for {formData.disability.diagnosticText}, <strong>after you got your disability rating</strong>.</p>
+  );
+};
+
+
+export const treatmentView = ({ formData }) => {
+  const { startTreatment, endTreatment, treatmentCenterName } = formData.treatment;
+  return (
+    <div>
+      <strong>{treatmentCenterName}</strong>
+      <p>{startTreatment} — {endTreatment}</p>
```

Comment:
Are neither of these required? If so, perhaps something like the conditional templating logic in EducationPeriodVew might also work here:
https://github.com/department-of-veterans-affairs/vets-website/blob/3166cadf727fbcff6fcfd181c04d435545bc4f96/src/js/vre/chapter31/components/EducationPeriodView.jsx#L3-L20

Reply:
Good call. I have zero clue about validation yet, though.